templates/_layouts/default.html

```diff
@@ -6,7 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>Multipass - Orchestrates virtual Ubuntu instances</title>
+  <title>{% block title %}{% endblock %} | Multipass - Orchestrates virtual Ubuntu instances</title>
```

Comment:
This is gonna look pretty funny if the `title` block isn't defined - the title will read:

>  | Multipass - Orchestrates virtual Ubuntu instances

Not sure what the best way around this is. Maybe we should change it to:

```
{% block title %}Orchestrate virtual Ubuntu instances{% endblock %} | Multipass
```

I'm not sure who to run that change by - maybe @pmahnke ?